frontend/js/components/dashboard/activityFeed.vue

```diff
@@ -71,14 +73,15 @@
       })
     },
     methods: {
+      getData(pageNumber) {
+        this.$http.get('admin?' + this.navFilters[this.navActive].slug + '=' + pageNumber).then(({data}) => {
```

Comment:
@ifox do we have a helper to get the "admin" url?

Reply:
Could we use the document location here?

Reply:
Decided to pass the base url from the backend to ensure it is correct. https://github.com/area17/twill/pull/2078/commits/92de5fc12f580c115eb3191ac85d6595a85cdbc3